tests/profiles-enabled/test_shipments.py

```diff
@@ -317,7 +317,6 @@ def test_add_tracking_data(self):
             data = response.json()['data']
             self.assertTrue(isinstance(data['features'], list))
             self.assertEqual(data['features'][0]['geometry']['type'], 'Point')
-            self.assertEqual(data['features'][0]['properties']['has_gps'], True)
```

Comment:
Since `has_gps` doesn't work with GeoSerializer, let's remove it from the TrackingData model altogether. It's not actually in the trackingdata schema, and I don't think it really adds anything that the `source` property doesn't already tell us.